lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/validation/JarValidator.java

```diff
@@ -223,4 +229,44 @@ private URI validateRedirectUri(JWTClaimsSet claimsSet, String clientId)
                             "Failed to parse JWT claim set in order to access redirect_uri claim"));
         }
     }
+
+    private void validateInheritedIdentityJwtClaim(JWTClaimsSet claimsSet)
```

Comment:
It feels a bit unsatisfactory to convert the claims object and perform almost all the same checks twice, but it looks ugly to avoid :(